app/middlewares/users/verify_signs_up.js

```diff
@@ -0,0 +1,40 @@
+const { badRequest_Error, defaultError } = require('../../errors');
+const logger = require('../../logger');
+
+const { encryptar } = require('../../helpers/utils');
+
+exports.verifyEmail = (req, res, next) => {
+  const { email } = req.body;
```

Comment:
The other fields still need to be validated remember that you can validate the schema with express-validator